projects/Mallard/src/App.tsx

```diff
@@ -44,6 +46,12 @@ fetchCacheClear().then((weOk: boolean) => {
     }
 })
 
+NetInfo.isConnected.addEventListener('connectionChange', isConnected => {
```

Comment:
Might be better to use the hook here: https://github.com/react-native-community/react-native-netinfo#usenetinfo

Reply:
@jimhunty Why?

Reply:
Otherwise there are two listeners. The hook already does the work. Perf improvement.

Reply:
@jimhunty So I can use the hook directly here?